rds-instance-full/main.tf

```diff
@@ -40,6 +40,29 @@ module "rds_security_group" {
   ingress_rules       = ["${var.ingress_rule}"]
 }
 
+module "custom_security_group" {
```

Comment:
Replace `ecs` in all arguments to this module with something like `custom`.

Word "custom" is more suitable for this module.

Reply:
All `ecs` references got removed